ze/zeinterval_callbacks.cpp.erb

```diff
@@ -293,6 +346,20 @@ static void zeinterval_<%= dbt_event.name %>_callback(
      int64_t ns_from_origin;
      bt_clock_snapshot_get_ns_from_origin(bt_clock, &ns_from_origin);
      create_and_enqueue_frequency_message(hostname.c_str(), process_id, thread_id, (uintptr_t)hDevice, domain, ns_from_origin, frequency);
+  <% elsif dbt_event.name_unsanitized == "lttng_ust_ze_sampling:computeEngine" %>
+     const hostname_t   hostname   = borrow_hostname(bt_evt);
+     const process_id_t process_id = 0;
+     const thread_id_t  thread_id  = 0;
+     int64_t ns_from_origin;
+     bt_clock_snapshot_get_ns_from_origin(bt_clock, &ns_from_origin);
+     create_and_enqueue_computeEU_message(hostname.c_str(), process_id, thread_id, (uintptr_t)hDevice, subDevice, activeTime, ns_from_origin);
```

Comment:
Why does we pass `process_id` and `thread_id`  to the `create_and_enqueu` is they are always 0?

Reply:
These parameters are part of the common_context in the interval model.